SharedLibs/TileMapHelper/src/model/TileMapModel.ts

```diff
@@ -1078,11 +1079,16 @@ export class TileDefinition {
    * Add a polygon for the collision layer
    * @param tag The tag to allow collision layer filtering.
    * @param polygon The polygon to use for collisions.
+   * @param hasFullHitBox Set to `true` when the hitBox cover the whole tile.
    */
-  addHitBox(tag: string, polygon: PolygonVertices): void {
+  addHitBox(
+    tag: string,
+    polygon: PolygonVertices,
+    hasFullHitBox: boolean
+  ): void {
```

Comment:
Idea: I wonder if this should have been a addFullTileHitBox(tag: string) that would have set the polygon itself?

Reply:
It seems complicated because the TileDefinition doesn't have the knowledge about the tile size.